scripts/package_building/build_artifacts.sh

```diff
@@ -529,12 +532,12 @@ function patch_kernel() {
         url=${patch#*//}
         case $protocol in
             http|https)
-                curl -s "$patch" | patch -f -p1 > /dev/null
+                curl -s "$patch" | git am > /dev/null
```

Comment:
we want the failure messages from this command

do this for both:
git_patch_msg=$(curl -s "$patch" | git am 2>&1)

Reply:
at line 544 put then echo "Patch failed with error message $git_patch_msg"